content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
+
+## Security
+
+- **Added** Enabled the workload cert rotate automatically in VM
+ ([Issue #24554](https://github.com/istio/istio/issues/24554))
+
+- **Added** support for creation of CSRs using ECC based certificates.
+ ([Issue #23226](https://github.com/istio/istio/issues/23226))
+
+- **Added** SDS support for Client Certificate and CA certificate used for TLS/mTLS Origination from Egress Gateway.
+  ([Issue #14039](https://github.com/istio/istio/issues/14039))
```

Comment:
@JimmyCYJ @nschhina @myidpt Do we have a istio.io doc for this new feature? If so, let's link it here.

Reply:
Yes we do, it's a "preliminary.istio.io" link right now since istio.io hasn't switched, can you point me to an example on how we add doc refs? The doc is here https://preliminary.istio.io/latest/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/ feel free to add it if you know how

Reply:
This should go into traffic management as its in destination rule. Please clearly mention the DestinationRule aspect.

Reply:
@jwendell can you move this to Traffic mgmt and add `using DestinationRule.` to the end of the sentence? Thanks!

Reply:
Done.

Reply:
It is a little misleading. Why is SDS support on Egress part of traffic management?